docs/getting-started-guides/clc.md

```diff
@@ -0,0 +1,317 @@
+```
+
+## Cluster Expansion
+
+To expand an existing Kubernetes cluster, run the add-kube-node.sh
+script. A complete list of script options and some examples are listed below.
+This script must be run from the same hose that created the cluster (or a host
+that has the cluster artifact files stored in ~/.clc_kube/$cluster_name).
+
+```
+cd ./adm-kubernetes-on-clc
+bash add-kube-node.sh -c="name_of_kubernetes_cluster" -m=2
+```
+
+#### Cluster Expansion: Script Options
+
+```
+Usage: add-kube-node.sh [OPTIONS]
+Create servers in the CenturyLinkCloud environment and add to an
+existing CLC kubernetes cluster
+
+Environment variables CLC_V2_API_USERNAME and CLC_V2_API_PASSWD must be set in
+order to access the CenturyLinkCloud API
+
+     -h (--help)                   display this help and exit
+     -c= (--clc_cluster_name=)     set the name of the cluster, as used in CLC group names
+     -m= (--minion_count=)         number of kubernetes minion nodes to add
+
+```
+
+## Cluster Deletion
+There are two ways to delete an existing cluster:
+
+1) Use our python script:
+
+```
+python delete_cluster.py --cluster=clc_cluster_name --datacenter=DC1
+
+```
+
+2) Use the CenturyLink Cloud UI. To delete a cluster, log into the CenturyLink
+Cloud control portal and delete the parent server group that contains the
+Kubernetes Cluster. We hope to add a scripted option to do this soon.
+
+## Examples
+Create a cluster with name of k8s_1, 1 master node and 3 worker minions (on physical machines), in VA1
+
+```
+ bash kube-up.sh --clc_cluster_name=k8s_1 --minion_type=bareMetal --minion_count=3 --datacenter=VA1
+```
+
+Create a cluster with name of k8s_2, an ha etcd cluster on 3 VMs and 6 worker minions (on VMs), in VA1
+
+```
+ bash kube-up.sh --clc_cluster_name=k8s_2 --minion_type=standard --minion_count=6 --datacenter=VA1 --etcd_separate_cluster=yes
+```
+
+Create a cluster with name of k8s_3, 1 master node, and 10 worker minions (on VMs) with higher mem/cpu, in UC1:
+
+```
+  bash kube-up.sh --clc_cluster_name=k8s_3 --minion_type=standard --minion_count=10 --datacenter=VA1 -mem=6 -cpu=4
+```
+
+
+
+## Cluster Features and Architecture
+We configue the Kubernetes cluster with the following features:
+
+* KubeDNS: DNS resolution and service discovery
+* Heapster/InfluxDB: For metric collection. Needed for Grafana and auto-scaling.
+* Grafana: Kubernetes/Docker metric dashboard
+* KubeUI: Simple web interface to view kubernetes state
+* Kube Dashboard: New web interface to interact with your cluster
+
+We use the following to create the kubernetes cluster:
+
+* Kubernetes 1.1.7
+* Unbuntu 14.04
+* Flannel 0.5.4
+* Docker 1.9.1-0~trusty
+* Etcd 2.2.2
+
+## Optional add-ons
+
+* Logging: We offer an integrated centralized logging ELK platform so that all
+  kubernetes and docker logs get sent to the ELK stack. To install the ELK stack
+  and configure kubernetes to send logs to it, follow this documentation: [log
```

Comment:
rather than "this documentation", change link text to "follow [the log aggreggation documentation]"